recipes/mpg123/all/test_package/CMakeLists.txt

```
@@ -1,10 +1,7 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package C)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES C)
 
 find_package(mpg123 REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
-target_link_libraries(${PROJECT_NAME} PRIVATE MPG123::libmpg123)
+target_link_libraries(${PROJECT_NAME} PRIVATE MPG123::libmpg123 MPG123::libout123 MPG123::libsyn123)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE MPG123::libmpg123)
```

Here I think `MPG123::libmpg123` alone is correct - they're 3 independent libraries (I can't see any dependencies between them), with 3 distinct header files - if the `test_package.c` file is only calling functionality from `mpg123.h` then this one suffices. If we add more libraries, on newer Linux distros `-as-needed` is passed and they will be eventually ignored, and on macOS the test executable will require the other two libraries at runtime (unnecessarily).